internal-client/src/main/java/org/apache/uniffle/client/impl/grpc/ShuffleServerGrpcNettyClient.java

```diff
@@ -283,7 +283,8 @@ public RssGetInMemoryShuffleDataResponse getInMemoryShuffleData(
     switch (rpcResponse.getStatusCode()) {
       case SUCCESS:
         LOG.info(
-            "GetInMemoryShuffleData from {}:{} for {} cost {} ms",
+            "GetInMemoryShuffleData size:{}(bytes) from {}:{} for {} cost:{}(ms)",
+            getMemoryShuffleDataResponse.body().size(),
```

Comment:
Could the body() be null?

Reply:
They will never be null. `GetLocalShuffleDataResponse`, `GetLocalShuffleIndexResponse` and `GetMemoryShuffleDataResponse` will always decode a ByteBuf. You can see the second input param of `decode` method is `true`.

The code in `org.apache.uniffle.common.netty.protocol.Message` will ensure this:
```
public static Message decode(Type msgType, ByteBuf in) {
  switch (msgType) {
    case RPC_RESPONSE:
      return RpcResponse.decode(in, false);
    case SEND_SHUFFLE_DATA_REQUEST:
      return SendShuffleDataRequest.decode(in);
    case GET_LOCAL_SHUFFLE_DATA_REQUEST:
      return GetLocalShuffleDataRequest.decode(in);
    case GET_LOCAL_SHUFFLE_DATA_RESPONSE:
      return GetLocalShuffleDataResponse.decode(in, true);
    case GET_LOCAL_SHUFFLE_INDEX_REQUEST:
      return GetLocalShuffleIndexRequest.decode(in);
    case GET_LOCAL_SHUFFLE_INDEX_RESPONSE:
      return GetLocalShuffleIndexResponse.decode(in, true);
    case GET_MEMORY_SHUFFLE_DATA_REQUEST:
      return GetMemoryShuffleDataRequest.decode(in);
    case GET_MEMORY_SHUFFLE_DATA_RESPONSE:
      return GetMemoryShuffleDataResponse.decode(in, true);
    default:
      throw new IllegalArgumentException("Unexpected message type: " + msgType);
  }
}
```

Reply:
OK.